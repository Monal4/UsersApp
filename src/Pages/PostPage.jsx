import React, { useState, useEffect } from 'react';
import axios from 'axios';
import UserPost from '../Components/Users-post-component/userPost.component'

const PostPage = (props) => {
    
    const[data, setData] = useState([]);

    async function report() {
        const response = await axios(`https://jsonplaceholder.typicode.com/posts?userId=`+ props.match.params.userId);
        setData(await response.data);
    }

    useEffect(() => {
        report();
    },[]);

    return(
        <UserPost post={data}/>
    )
}

export default PostPage;