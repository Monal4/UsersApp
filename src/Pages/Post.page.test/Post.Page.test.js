import React from 'react';
import { shallow } from "enzyme";
import '../../setup';
import PostPage from "../PostPage";

describe('Post Page Component test', () => {
    
    it('is a snapshot for PostPage', () => {
        expect( shallow(<PostPage/>) ).toMatchSnapshot();
    });

});