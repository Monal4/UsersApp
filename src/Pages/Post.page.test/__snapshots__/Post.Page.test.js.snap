// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post Page Component test is a snapshot for PostPage 1`] = `ShallowWrapper {}`;
